src/stylesheets/base/_typography.scss

```diff
@@ -1,88 +1,110 @@
 // UNIVERSAL
+
 .hdr-text {
-  @include text( $primary-font, normal, 1.4rem, $color-dark-green );
+  @include text($primary-font, normal, 1rem, $color-dark-green);
   text-transform: uppercase;
-  letter-spacing: .2em;
-
+  letter-spacing: 0.2em;
+  text-align: center;
   @media (min-width: 550px) {
     font-size: 1.1rem;
   }
 }
 
 .intro-text {
-  @include text( $secondary-font, normal, 1.8rem, $color-grey );
+  @include text($secondary-font, normal, 1.8rem, $color-grey);
   text-align: center;
-
   @media (min-width: 550px) {
     font-size: 2.1rem;
     text-align: left;
   }
 }
 
 .context-text {
-  @include text( $primary-font, normal, 1.4rem, $color-grey );
+  @include text($primary-font, normal, 1.5rem, $color-grey);
   line-height: 2rem;
   text-align: center;
-  letter-spacing: .05em;
-
+  letter-spacing: 0.05em;
   @media (min-width: 550px) {
     text-align: left;
     letter-spacing: none;
   }
 }
-
 // NAVIGATION
+
 .nav__wrapper a {
-  @include text( $primary-font, normal, 1.1rem, $color-mint );
+  @include text($primary-font, normal, 1rem, $color-mint);
   text-transform: uppercase;
   text-decoration: none;
-
   @media (min-width: 550px) {
```

Comment:
leave an extra line before the media query. You get the idea... I'll stop commenting on it